python/lsst/analysis/tools/analysisPlots/analysisPlots.py

```diff
@@ -183,3 +189,60 @@ def setDefaults(self):
         self.produce.yAxisLabel = "Dec. (degrees)"
         self.produce.zAxisLabel = "Ap 12 - PSF [mag]"
         self.produce.plotOutlines = False
+
+
+class StellarPhotometricRepeatabilityPlot(AnalysisPlot):
+    fluxType: str = "psfFlux"
+
+    def setDefaults(self):
+        super().setDefaults()
+
+        # Apply per-source selection criteria
+        self.prep.selectors.bandSelector = BandSelector()
+
+        # Compute per-group quantities
+        self.process.buildActions.perGroupSn = PerGroupStatistic()
+        self.process.buildActions.perGroupSn.buildAction = Sn(fluxType=f"{self.fluxType}")
+        self.process.buildActions.perGroupSn.func = "median"
+        self.process.buildActions.perGroupExtendedness = PerGroupStatistic()
+        self.process.buildActions.perGroupExtendedness.buildAction.vectorKey = "extendedness"
+        self.process.buildActions.perGroupExtendedness.func = "median"
+        self.process.buildActions.perGroupCount = PerGroupStatistic()
+        self.process.buildActions.perGroupCount.buildAction.vectorKey = f"{self.fluxType}"
+        self.process.buildActions.perGroupStdev = PerGroupStatistic()
+        self.process.buildActions.perGroupStdev.buildAction = MagColumnNanoJansky(
+            vectorKey=f"{self.fluxType}"
+        )
+        self.process.buildActions.perGroupStdev.func = "std"
+
+        # Filter on per-group quantities
+        self.process.filterActions.perGroupStdevFiltered = DownselectVector(vectorKey="perGroupStdev")
+        self.process.filterActions.perGroupStdevFiltered.selectors.count = ThresholdSelector(
+            vectorKey="perGroupCount",
+            op="ge",
+            threshold=3,
+        )
```

Comment:
When I tried to execute the plotting portion of the code, it gave an error on line 220 that "AttributeError: 'DownselectVector' object has no attribute 'selectors'". I think you either need to create an empty dict (or list?) before that line, or something similar to give it a "selectors" attribute.

Reply:
I am removing the plotting capability from this PR.